corehq/apps/analytics/static/analytix/js/gtm.js

```diff
@@ -0,0 +1,68 @@
+"use strict";
```

Comment:
is this copied from gtm?

Reply:
No, this script does the job of loading the analytics script and passing the desired user info [here](https://github.com/dimagi/commcare-hq/blob/6cf6903589b9c436d2f14547c4276f0493d0956b/corehq/apps/analytics/static/analytix/js/gtm.js#L51).
This is same approach used for loading other analytics tooling script.